static/bidder-info/epsilon.yaml

```diff
@@ -13,6 +15,6 @@ capabilities:
       - video
 userSync:
   redirect:
-    url: "https://prebid-match.dotomi.com/match/bounce/current?version=1&networkId=72582&rurl={{.RedirectURL}}"
+    url: "https://prebid-match.dotomi.com/match/bounce/current?version=1&networkId=72582&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&rurl={{.RedirectURL}}"

```

Comment:
Curious, why are you no longer checking for privacy policies?

Reply:
@SyntaxNode I'm not sure what you mean, what do you believe has changed in this pull request?